From bce7a403e49ae97797010f0e05d83437867eeec9 Mon Sep 17 00:00:00 2001
From: Leo Wang <leo.jt.wang@fii-foxconn.com>
Date: Tue, 10 Jun 2025 22:37:04 +0800
Subject: [PATCH] ARM: dts: aspeed: clemente: Correct I2C addresses

Corrects the Interposer FRU EEPROM and fan control 3 devices address.
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-clemente.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-clemente.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-clemente.dts
index 82a3f6f5b991..651f9494b906 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-clemente.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-clemente.dts
@@ -576,9 +576,9 @@ channel@6 {
 	};
 
 	// Module 1
-	fanctl3: fan-controller@27{
+	fanctl3: fan-controller@2f{
 		compatible = "maxim,max31790";
-		reg = <0x27>;
+		reg = <0x2f>;
 		#address-cells = <1>;
 		#size-cells = <0>;
 		channel@5 {
@@ -722,9 +722,9 @@ i2c1mux0ch5: i2c@5 {
 			reg = <0x5>;
 
 			// Interposer FRU EEPROM
-			eeprom@51 {
+			eeprom@54 {
 				compatible = "atmel,24c64";
-				reg = <0x51>;
+				reg = <0x54>;
 			};
 
 			// Interposer TEMP SENSOR
-- 
2.43.0

